app/schema/answer.py

```diff
@@ -100,3 +100,14 @@ def mandatory_error(self, state):
         state.errors = []
         state.errors.append(self.questionnaire.get_error_message('MANDATORY', self.id))
         state.is_valid = False
+
+    @staticmethod
+    def _valid_option_selected(state_input, options):
```

Comment:
Cool, other than that it all looks good and you've made a good effort in backfilling the checkbox tests!